Index: edit.php
===================================================================
--- edit.php	(revision 24242)
+++ edit.php	(working copy)
@@ -712,6 +712,11 @@
             {
                 switch( $operationResult['status'] )
                 {
+					case eZModuleOperationInfo::STATUS_REPEAT:
+                    {
+                        eZContentOperationCollection::setVersionStatus( $object->attribute( 'id' ),
+                            $version->attribute( 'version' ), eZContentObjectVersion::STATUS_REPEAT );
+                    }
                     case eZModuleOperationInfo::STATUS_HALTED:
                     {
                         if ( isset( $operationResult['redirect_url'] ) )
